packages/runtime/src/server.ts

```diff
@@ -36,6 +37,15 @@ export interface ServerConfig {
   environment?: Environment | string;
 }
 
+export const captureException = (exception: any) => {
+  logger.error(serializeError(exception));
+  if(process.env.SENTRY_DSN) {
+    Sentry.captureException(exception);
+  } else {
+    logger.info('captureException called but sentry is not set up.');
```

Comment:
```suggestion
    logger.debug('captureException did not create an alert');
```
🤔 I don't want this message to look like something is wrong when devs are running locally.